### PR TITLE
chore(ci): fix workflows

### PR DESCRIPTION
**Pull request by pallabmaiti**
# Description

Fix Workflows.

### CODE REVIEW

File: Rudder.podspec [hunk 1]
```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
 
   s.homepage         = "https://github.com/rudderlabs/rudder-sdk-ios"
   s.license          = { :type => "Apache", :file => "LICENSE" }
-  s.author           = { "RudderStack" => "arnab@rudderlabs.com" }
+  s.author           = { "RudderStack" => "sdk-accounts@rudderlabs.com" }
```
Comment (saikumarrs) [inline Rudder.podspec:15]: Should this be `sdk-accounts@rudderstack.com`?

Comment (pallabmaiti) [reply Rudder.podspec]: Yes. Corrected.

File: Rudder.podspec [hunk 2]
```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
 
   s.homepage         = "https://github.com/rudderlabs/rudder-sdk-ios"
   s.license          = { :type => "Apache", :file => "LICENSE" }
-  s.author           = { "RudderStack" => "arnab@rudderlabs.com" }
+  s.author           = { "RudderStack" => "sdk-accounts@rudderstack.com" }
```
Comment (bardisg) [inline Rudder.podspec:15]: sdk-accounts is the user for third party tools accounts. sdk@rudderstack.com is the one for the team group account

Comment (pallabmaiti) [reply Rudder.podspec]: Updated.

**Review by bardisg — APPROVED** [review]

**Pull request merged**